### DISCUSSION TITLE
请问一下quora的广告可以去掉吗？

### DISCUSSION THREAD

**Discussion by haorened** [blackmatrix7/ios_rule_script#851] [Ideas]
不小心点到或者混入答案感觉还是有点烦

**Comment by blackmatrix7** [comment]
这个仓库主要满足我个人需求，没有推广和引流的需求，所以通常不接受任何脚本的定制。

不过，Quora是个例外。

我正在使用这个App，并且有个未公开的去广告脚本。

现在公开了：

https://github.com/blackmatrix7/ios_rule_script/tree/master/script/quora

**Comment by haorened** [comment]
谢谢大佬
